lib/ansible/modules/files/copy.py

```diff
@@ -78,7 +78,7 @@
     description:
       - If C(no), it will search for I(src) at originating/master machine.
       - If C(yes) it will go to the remote/target machine for the I(src). Default is C(no).
-      - Currently I(remote_src) does not support recursive copying.
+      - I(remote_src) supports recursive copying in version 2.7 and higher.
```

Comment:
- I(remote_src) supports recursive copying as of version 2.8.

Reply:
concise statement!